content/docs/tutorials/azure/azure-ts-appservice-springboot.md

```diff
@@ -2,7 +2,7 @@
 title: "Deploy a Spring Boot App using Jenkins and Pulumi"
 ---
 
-<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-springboot" target="_blank">
+<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-springboot/infrastructure" target="_blank">
```

Comment:
I updated the button on the README.md for this example so that it would actually work with the Pulumi program being in a subdir.

Reply:
Thanks!